beta-src/src/state/game/game-api-slice.ts

```diff
@@ -395,14 +447,14 @@ const gameApiSlice = createSlice({
                 [orderToUpdate.id]: {
                   saved: false,
                   update: {
-                    type: "Hold",
+                    type: phase === "Retreats" ? "Disband" : "Hold",
                     toTerrID: null,
                   },
                 },
               });
             }
           }
```

Comment:
shouldn't this be uppercase "Disband" like you have on L450

Reply:
No this is checking phase which is not all caps